tests/api/admin/controller/test_report.py

```diff
@@ -22,7 +23,9 @@ def report_fixture(
 
 
 class TestReportController:
-    def test_generate_inventory_report(self, report_fixture: ReportControllerFixture):
+    def test_generate_inventory_and_hold_reports(
+        self, report_fixture: ReportControllerFixture, celery_fixture: CeleryFixture
```

Comment:
Minor: This works, but in the other controller tests I've written so far, I tried to isolate the controller from the queue, by mocking out the task. This way we don't have the bring `celery_fixture` into the controller tests. 

Something like: https://github.com/ThePalaceProject/circulation/blob/8e976536b16a2930c8032ead7a8dde5d34a0e3a2/tests/api/admin/controller/test_collections.py#L711-L713

Mocking out celery in web controllers like this aligns with what the celery docs suggest for testing strategies: https://docs.celeryq.dev/en/stable/userguide/testing.html#tasks-and-unit-tests